sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/SessionCatalog.scala

```diff
@@ -121,8 +123,11 @@ class SessionCatalog(
     currentDb = db
   }
 
-  def getDefaultDBPath(db: String): String = {
-    System.getProperty("java.io.tmpdir") + File.separator + db + ".db"
+  def createDatabasePath(dbName: String, path: Option[String]): String = {
```

Comment:
Can you define what this function does? Most other functions they are obvious. It is not clear what this is doing. Is this one creating some new path?